app/models/flight.rb

```diff
@@ -14,29 +14,51 @@ class Flight < ActiveRecord::Base
   validates :available_seats, presence: true
   validates :price, presence: true
 
+  scope(
+    :not_departed,
+    lambda do
+      where("departure_datetime > ?", Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.